napari/components/_dims/_constants.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class DimsMode(Enum):
```

Comment:
not really a huge fan of creating a whole new file just for this but ok

Reply:
fair, we can revisit if others feel strongly too

Reply:
I’m ok with this being in its own file.

Reply:
To me it makes sense to put that in the Dims model, since this is only used in that context, and is very short... If it gets more complicated, longer... then we could make it its own file. At the same time I don't have a strong opinion -- as I see that this is at least in the _dims package...

Reply:
They are used in both the `model` and `view` files. I'm going to leave separate for now